### PR TITLE
Enable counterexamples

### PR DESCRIPTION
**Pull request by typerSniper**

### CODE REVIEW

**Comment by typerSniper** [comment]
@atomb

File: Sources/SymDiff/source/RVTCheck.cs [hunk 1]
```diff
@@ -480,20 +480,18 @@ public static int RVTRunVerificationTask(SDiff.VerificationTask vt, VC.Condition
 
             vcgen = BoogieVerify.InitializeVC(newProg);
             //SDiff.Boogie.Process.ResolveAndTypeCheck(newProg, "");
-            var newDict = SDiff.Boogie.Process.BuildProgramDictionary(newProg.TopLevelDeclarations.ToList());
             //newEq = (Implementation)newDict.Get(vt.Eq.Name + "$IMPL");
-
+            var newDict = SDiff.Boogie.Process.BuildProgramDictionary(newProg.TopLevelDeclarations.ToList());
```
Comment (atomb) [inline Sources/SymDiff/source/RVTCheck.cs:484]: Is this change meaningful?

File: Sources/SymDiff/source/BoogieVerify.cs [hunk 2]
```diff
@@ -835,6 +783,35 @@ static public void ReplaceInFile(string filePath, string searchText, string repl
             reader.Close();
             writer.Close();
         }
+
+        public static string RunBoogieCommand(string fileName){
```
Comment (atomb) [inline Sources/SymDiff/source/BoogieVerify.cs]: Is this method used anywhere?

File: Sources/SymDiff/source/RVTCheck.cs (hunk 1 above)
Comment (typerSniper) [reply Sources/SymDiff/source/RVTCheck.cs:484]: No its not. I was trying to fix something for RVTCheck and this stayed

File: Sources/SymDiff/source/BoogieVerify.cs (hunk 2 above)
Comment (typerSniper) [reply Sources/SymDiff/source/BoogieVerify.cs]: Removed :)

**Review by atomb — APPROVED** [review]

**Pull request merged**